R/smc_post_processing_functions.R

```diff
@@ -429,6 +431,7 @@ compute_posterior_intervals_rho <- function(output, nmc, burnin, verbose=FALSE)
 #'
 compute_rho_consensus <- function(output, nmc, burnin, C, type, verbose=FALSE) {
   n_items <- dim(output)[2]
+  N <- dim(output)[1]
   smc_plot <- smc_processing(output = output)
 
   iteration <- array(rep((1:N), n_items)) # ASK: what is N? It is out of scope. Gather from other objects or add as argument?
```

Comment:
This is a typo - my mistake! I think N needs to be replaced with nmc.
N is the number of particles in SMC, but I use this function when running BayesMallows MCMC and SMC.
So when I run SMC and use this function I use the following input compute_rho_consensus(.... nmc = N ...).

Reply:
Resolved by [comment below](https://github.com/ocbe-uio/BayesMallows/pull/79#discussion_r609680392).